### ISSUE TITLE
Widget Visibility: options not available when using the Accessibility mode

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#681]
Steps to reproduce:
1. Activate Jetpack and the Widget Visibility module
2. Go to Appearance > Widgets in your dashboard
3. Click on "Screen options"
4. Enable the accessibility mode
5. Click on "Edit" to edit a widget
   - The widget visibility options are nowhere to be found.

**Comment by chrisdc** [comment]
I think @ivankristianto is right in that there needs to be a hook in widgets.php before we can add anything in accessibility mode. Might it be worth opening a ticket on the main WordPress trac?

**Comment by kraftbj** [comment]
Adding a ticket in Core makes sense if a hook is needed. Before going too deep down that path, assuming a hook in Core was present, how would we display all of the various options without JS?

**Comment by chrisdc** [comment]
I feel like I must be missing something better, but the best option I can see right now would be to make the user select the rule type (category, author etc.) then click add to reload the page with a populated dropdown in place. The add and delete links could both be handled by linking back to admin/widgets.php with a custom query string, and our method could check for this in $_POST, and act accordingly before rendering the form on each reload. This would also mean saving the settings on every click of add or delete.

**Comment by dkager** [comment]
I ran into this today. Would be great if it could be picked up for v4.1. :)
By the way, the JavaScript-based rule editor is perfectly accessible to screen reader software for the visually impaired. (Arguably, so is dragging and dropping, though it's not good UX.) I can't speak for other disabilities, but completely barring JS seems a bit harsh.

**Comment by chickenn00dle** [comment]
The issue persists as of Jetpack 7.8
